workspaces/confluence/packages/backend/src/index.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Hi!
+ *
+ * Note that this is an EXAMPLE Backstage backend. Please check the README.
+ *
+ * Happy hacking!
+ */
+
+import { createBackend } from '@backstage/backend-defaults';
+
+const backend = createBackend();
+
+backend.add(import('@backstage/plugin-app-backend/alpha'));
+backend.add(import('@backstage/plugin-proxy-backend/alpha'));
+backend.add(import('@backstage/plugin-scaffolder-backend/alpha'));
+backend.add(import('@backstage/plugin-techdocs-backend/alpha'));
```

Comment:
Do we actually need these? Proxy, Scaffolder, TechDocs?

Reply:
No. But i observed that other plugins are not remove default `create-app` imports.
(e.g. `azure-devops`, `jenkins`, `linguist`)

Reply:
keeping `scaffoldr` offers the opportunity to move the code of the [confluence-to-mardown](https://github.com/backstage/backstage/tree/master/plugins/scaffolder-backend-module-confluence-to-markdown) action

Reply:
Sure, and we should probably remove it there as well. Just calling it out now as it's pretty easy to do, right?

Reply:
right :)

Reply:
I think it would be better not to do that all in this PR. There's probably work that needs to be coordinated in the Backstage repo as well to make this transfer smooth.

Reply:
Apologies if I gave the impression that is something you should do now.

Reply:
no problem, remove the sources

Reply:
Hello @awanlin , I'm going to remove `plugin-proxy-backend`, `plugin-scaffolder-backend` and `plugin-techdocs-backend` plugins.
Should i remove their frontends as well ?

Reply:
Yes, please.

Reply:
Frontend plugins are removed.